ug/instructors.md

```diff
@@ -56,3 +77,12 @@ The repositories used for initial bug reporting will be created automatically by
   This must be a public repo with write access given to students.
 * A repo to hold bugs for tutors to moderate e.g., `module-org/pe-moderation`<br>
   This can be a private repo with write access given to tutors.
+  
+
+## Creating a session link for students
+
+A list of profiles can be modified in the Profile repository. 
+
+To create a session link, add the name of the session after this url: `https://catcher-org.github.io/CATcher/?session=`. This profile name must correspond exactly to the profile name in Profile repository. 
+
+Students can access the session link to apply the corresponding session to the field if the profile name is valid from the list of profiles. Students can then click the Submit button to enter the session specified without selecting from the dropdown.
```

Comment:
```suggestion
If the profile name is valid (found in profile.json), the corresponding session will automatically be selected. Students can then click the Submit button to enter the session specified without selecting from the dropdown.
```
Not sure if this will sound better.

Reply:
Rewrote to ```
If the profile name is valid, students can access the link to select the corresponding session automatically. Students can then click the Submit button to enter the intended session without selecting from the dropdown.
```